nx.json

```diff
@@ -4,6 +4,7 @@
     "default": {
       "runner": "@nrwl/nx-cloud",
       "options": {
+        "accessToken": "ZTMzMDk4MmEtNjMyOC00ZjFiLWI5MTktYTc1ODhiOTI0OTJifHJlYWQ=",
```

Comment:
This is a read-only token to read the build cache. We are hard coding it to keep the workflow simple.